enter.c

```diff
@@ -471,6 +471,7 @@ int enter(struct entry_settings *opts)
 			err(1, "chdir(\"/\")");
 		}
 	}
+	umask(0022);
```

Comment:
I don't think this is the right way to fix this problem.  The model of bst is _not_ that it provides a clean environment by default.  It literally does not clean the environment, for example, and it does not change UID or GID or CWD or anything else by default.  The model of bst is that it provides namespace isolation by default (for backward compatibility reasons when new namespaces become supported) but cleans nothing by default.

I am in favor of adding --umask=022 as a command-line option and calling umask() if passed, like we do for all other options.

Reply:
`--umask=022` works for me. That's definitely more consistent with the other features here